src/app/sb-admin/components/dashboard/dashboard.component.ts

```diff
@@ -1,46 +1,98 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
-import { MenuItem } from 'primeng/api';
-import { User } from '../../api/user';
-import { UserService } from '../../service/user.service';
-import { Subscription } from 'rxjs';
-import { LayoutService } from 'src/app/layout/service/app.layout.service';
+import { map } from 'rxjs';
+import { UserCountService } from '../../service/user-count.service';
+import { TenantDetails } from './tenantDetails';
+import $ from 'jquery';
 
 @Component({
-    templateUrl: './dashboard.component.html',
+  templateUrl: './dashboard.component.html',
 })
 export class DashboardComponent implements OnInit, OnDestroy {
 
-    items!: MenuItem[];
+  tenantDetail: TenantDetails[] = [];
 
-    users!: User[];
+  loading: boolean = true;
 
-    chartData: any;
+  tenantUserCountResponse: any[] = [];
 
-    chartOptions: any;
+  constructor(private userCountService: UserCountService) 
+  {}
 
-    subscription!: Subscription;
+  ngOnInit() {
+    $(document).ready(function () {
+      $("#eng").click(function () {
+        localStorage.setItem('lang', 'en');
+        document.location.reload();
+      });
+    });
 
-    constructor(private userService: UserService, public layoutService: LayoutService) {
-    }
+    this.getTenant().subscribe((data: any) => {
+      if (data && data.length > 0) {
+        this.tenantUserCountResponse = data;
+        this.getTenantUsercount(data);
+      }
+    });
+  }
+
+  //Get all tenant data
+  getTenant() {
+    const body = {
+      "request": {
+        "filters": {
+          "isRootOrg": true
+        },
+        "fields": [
+          "id",
+          "channel",
+          "orgName",
+          "externalId",
+          "isRootOrg"
+        ],
+        "sortBy": {
+          "createdDate": "Desc"
+        },
+        "limit": 1002
```

Comment:
@nidhikachhawa24 @poojakarma can you check this limit parameter to get tenant API? For me looks likes it is hardcoded.